### PR TITLE
Expand coverage

### PR DESCRIPTION
**Pull request by ashtul**

### CODE REVIEW

File: src/cuckoo.c [hunk 1]
```diff
@@ -374,6 +374,7 @@ size_t CuckooFilter_Compact(CuckooFilter *cf) {
     return ret;
 }
 
+/* CF.DEBUG uses another function
```
Comment (gkorland) [inline src/cuckoo.c]: @ashtul why do you keep dead code?

File: src/rm_cms.c [hunk 2]
```diff
@@ -34,9 +34,9 @@ static int GetCMSKey(RedisModuleCtx *ctx, RedisModuleString *keyName, CMSketch *
 
 static int CreateCMSKey(RedisModuleCtx *ctx, RedisModuleString *keyName, long long width,
                         long long depth, CMSketch **cms, RedisModuleKey **key) {
-    if (*key == NULL) {
+    /*if (*key == NULL) {
```
Comment (gkorland) [inline src/rm_cms.c]: @ashtul why do you keep dead code?

Comment (ashtul) [reply src/rm_cms.c]: Made some more cleanup.

File: src/cuckoo.c (hunk 1 above)
Comment (ashtul) [reply src/cuckoo.c]: It is an info function which doesn't hurt to keep for possible future use.

**Pull request merged**